.circleci/config.yml

```diff
@@ -108,6 +123,19 @@ workflows:
           name: "End-To-End Kubernetes 1.25"
           kind_node_image: "kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1"
           <<: *e2e_configuration
+      - rok8s/kubernetes_e2e_tests:
+          name: "Chainsaw End-To-End Kubernetes 1.23"
+          kind_node_image: "kindest/node:v1.23.13@sha256:ef453bb7c79f0e3caba88d2067d4196f427794086a7d0df8df4f019d5e336b61"
+          <<: *chainsaw_configuration
+      - rok8s/kubernetes_e2e_tests:
+          name: "Chainsaw End-To-End Kubernetes 1.24"
+          kind_node_image: "kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315"
+          <<: *chainsaw_configuration
+      - rok8s/kubernetes_e2e_tests:
```

Comment:
I think we should drop 1.23 and 1.24 and move to 1.25-1.27 instead.

Reply:
I agree, maybe in a follow up PR ? This is not related to this change.

Reply:
That's fine with me